internal/provisioner/fluentbit.go

```diff
@@ -80,17 +79,53 @@ func (f *fluentbit) NewHelmDeployment(logger log.FieldLogger) *helmDeployment {
 	if err != nil {
 		logger.WithError(err).Error("unable to lookup private zone name")
 	}
+
+	zoneID, err := f.awsClient.GetPrivateZoneID(logger)
+	if err != nil {
+		logger.WithError(err).Error("unable to find Zone ID")
+	}
+
+	tag, err := f.awsClient.GetTagByKeyAndZoneID(aws.DefaultAuditLogsCoreSecurityTagKey, zoneID, logger)
+	if err != nil {
+		logger.WithError(err).Error("failed to find Tag:AuditLogsCoreSecurity")
+	}
+	if tag == nil {
+		logger.Info("unable to find tag")
```

Comment:
Should this be a Warning? It seems a little strange that the error message on line 90 is the same as this one. I see the difference between the cases, but it seems a little odd to me that both errors are essentially the same (and handled the same way) but one is INFO level and the other is ERROR level. In my opinion both should be WARNING if they are severe enough to warrant the log message but inconsequential enough that execution continues.

Reply:
I think on line 90 it is an error from trying to retrieve the message, while on line 93 the message should be more informative that there wasn't any tag so "skipping setup" probably I should change the message of line 93. Does it make sense?

Reply:
>the message should be more informative that there wasn't any tag so "skipping setup"

Yeah this makes sense to me. Just adding that little extra context would be awesome.

Reply:
You are right! I need to change it with a better context! will add the skipping setup